docs/glossary.md

```diff
@@ -6,14 +6,6 @@
  the part of the Atomist platform that is operated by Atomist the company:
  the event hub, GraphQL endpoint, and web interface. [more info](developer/architecture.md)
 
-#### PushRule
-
-a specification for goals to set on a given push. [more info](developer/set-goals.md#pushrule)
```

Comment:
Not sure why these were here, but I alphabetically moved these closer to the `p`s. Maybe because capital letters are alphabetized ahead of lowercase ones?

Reply:
oh dear. This is alphabetized by the SDM, so I need to change that